relm4-macros/src/attrs.rs

```diff
@@ -1,31 +1,176 @@
-use proc_macro2::TokenStream as TokenStream2;
-use quote::ToTokens;
-use syn::{
-    parse::{Parse, ParseStream},
-    Result, Visibility,
-};
+use proc_macro2::Span;
+use syn::Error;
+use syn::Ident;
+use syn::Path;
+use syn::PathArguments;
+use syn::PathSegment;
+use syn::Result;
+use syn::Token;
+use syn::Visibility;
+use syn::parse::Parse;
+use syn::parse::ParseStream;
+use syn::punctuated::Punctuated;
+use syn::spanned::Spanned;
+use syn::token::Colon2;
```

Comment:
Maybe this can be written more compact. Like `syn::{Error, Ident, ...}`

Reply:
I'm using this syntax to reduce the amount of conflicts. 

I'll make it compact if you prefer